test/preset-icons.test.ts

```diff
@@ -17,9 +17,33 @@ describe('preset-icons', () => {
     ],
   })
 
+  const unoWithUnit = createGenerator({
+    presets: [
+      presetIcons({
+        unit: 'rem',
+        scale: 2,
+        customizations: {
+          iconCustomizer(collection, icon, props) {
+            if (collection === 'carbon' && icon === 'sun') {
```

Comment:
Update this to change the rest of icons and so this test allow us to check it is not overridden (once logic updated)

Reply:
Just negate the if condition